pgrx-sql-entity-graph/src/control_file.rs

```diff
@@ -83,13 +96,13 @@ impl ControlFile {
             relocatable: temp
                 .get("relocatable")
                 .ok_or(ControlFileError::MissingField { field: "relocatable" })?
-                == &"true",
+                == "true",
```

Comment:
`cargo clippy` got upset once I touched this file.  Same below on lines 103 and 105.